From 022448e914280859ff5b69f0ca511875a5c68e98 Mon Sep 17 00:00:00 2001
From: Dominik Haller <d.haller@phytec.de>
Date: Tue, 30 Aug 2022 14:08:26 +0200
Subject: [PATCH 1/3] Makefiles: Fix arch assignement

Add phytec toolchain name so the proper target architecture gets
selected.

Signed-off-by: Dominik Haller <d.haller@phytec.de>
---
 eurasia_km/eurasiacon/build/linux2/config/compiler.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/eurasiacon/build/linux2/config/compiler.mk b/eurasiacon/build/linux2/config/compiler.mk
index 1b5f0905f6e0..5f4fed7e6bb6 100644
--- a/eurasiacon/build/linux2/config/compiler.mk
+++ b/eurasiacon/build/linux2/config/compiler.mk
@@ -68,7 +68,7 @@ define calculate-compiler-preferred-target
   ifneq ($$(filter aarch64-%,$$($(1)_compiler_preferred_target)),)
    $(1)_compiler_preferred_target := aarch64-linux-gnu
   endif
-  ifneq ($$(filter arm-%-gnueabihf arm-oe-linux-gnueabi arm-poky-linux-gnueabi arm-yoe-linux-gnueabi arm-oe-linux-musleabi arm-poky-linux-musleabi arm-yoe-linux-musleabi,$$($(1)_compiler_preferred_target)),)
+  ifneq ($$(filter arm-%-gnueabihf arm-phytec-linux-gnueabi arm-oe-linux-gnueabi arm-poky-linux-gnueabi arm-yoe-linux-gnueabi arm-oe-linux-musleabi arm-poky-linux-musleabi arm-yoe-linux-musleabi,$$($(1)_compiler_preferred_target)),)
    $(1)_compiler_preferred_target := arm-linux-gnueabihf
   endif
   ifneq ($$(filter arm-%-gnueabi armv7a-cros-linux-gnueabi armv7hl-redhat-linux-gnueabi,$$($(1)_compiler_preferred_target)),)
--
2.25.1
